frontend/packages/console-app/src/components/nav/NavHeader.scss

```diff
@@ -1,5 +1,5 @@
 @import '~@patternfly/patternfly/sass-utilities/all';
-@import '../../style/vars';
+@import '../../../../../public/style/vars';
```

Comment:
I think we can import style sheets similarly to the way we import js packages:
```suggestion
@import '~@console/internal/style/vars';
```

Reply:
Thanks, fixed.